src/web/routes/application/register-form-routes.js

```diff
@@ -18,13 +18,28 @@ const handleOptionalMiddleware = (operation, fallback = middlewareNoop) => {
   return operation
 }
 
-const createRoute = (csrfProtection, steps, router) => (step) =>
+const getPreviousPage = (steps, step) => {
+  const index = steps.indexOf(step)
+
+  if (index === -1) {
+    throw new Error(`Unable to find ${step} in the list of steps`)
```

Comment:
Will this work for the confirmation page?

Reply:
This isn't used there.